notebooks/fixtures/config-exp/base-file.yaml

```diff
@@ -0,0 +1,28 @@
+user:
+  ACCOUNT: ""
+  MACHINE: ""
+workflow:
+  EXPTDIR: ""
+task_make_ics:
+  chgres_cube:
+    namelist:
+      update_values:
+        config:
+          cycle_day: !int "{{ cycle.strftime('%d') }}"
+          cycle_hour: !int "{{ cycle.strftime('%H') }}"
+          cycle_mon: !int "{{ cycle.strftime('%m') }}"
+          convert_atm: true
+          convert_nst: true
+          convert_sfc: true
+          data_dir_input_grid: "{{ task_make_ics.chgres_cube.rundir }}"
+          external_model: "{{ task_get_extrn_ics.EXTRN_MDL_NAME_ICS }}"
+          input_type: "gaussian_nemsio"
+          tg3_from_soil: false
+          tracers:
+            - sphum
+            - liq_wat
+          tracers_input:
+            - spfh
+            - clwmr
+          varmap_file: "{{ user.PARMdir }}/ufs_utils/varmap_tables/GFSphys_var_map.txt"
+    rundir: '{{ workflow.EXPTDIR }}/make_ics'
```

Comment:
I like the example, but I'm happy to hear other thoughts/concerns.

Reply:
Thanks, @maddenp-noaa I think these are all good suggestions and commit [a298427](https://github.com/ufs-community/uwtools/pull/653/commits/a2984272b8548fb8830901a887b45a11f49438d6) attempts to address each one. 

I had a bit of trouble naming this notebook and maybe in light of Paul's point about the notebook demonstrating a step in a larger experiment and the removal of the `configure_experiment()` function, the name may not make sense. Any suggestions for a different name would be welcome.

Reply:
Maybe something like _Building and Validating an Experiment Configuration_?

Would we like to have a `cookbooks/` subdirectory? If so, we could probably remove `cookbook` from the filename, as it'd be implied by the directory name.

Reply:
I like _Building and Validating an Experiment Configuration_ as the name here. I tried to move this notebook to a `cookbooks/` directory, but ran into a problem with unit testing because of the change. Testbook seems to run the notebook in whichever directory that pytest is called from, so the relative paths I use in the notebook wouldn't work either when called from the `notebooks/` or `cookbooks/` directory. I'm not sure absolute paths could work either since they would be system-dependent. For now I shortened the name of the notebook to end with `-cb` so at least the name isn't quite so long.